HopsanGenerator/src/generators/HopsanFMIGenerator.cpp

```diff
@@ -429,11 +470,11 @@ bool HopsanFMIGenerator::generateFromFmu1(const QString &rFmuPath, const QString
 
     QString setPars;
     QString temp = extractTaggedSection(fmuComponentCode, "setpars");
-    for(int i=0; i<parNames.size(); ++i)
+    for(const auto &par : pars)
     {
         QString tempVar = temp;
-        tempVar.replace("<<<vr>>>", parRefs.at(i));
-        tempVar.replace("<<<var>>>", parVars.at(i));
+        tempVar.replace("<<<vr>>>", par.fmuVr);
+        tempVar.replace("<<<var>>>", par.fmuVr);
```

Comment:
Should this really be par.fmuVr in both cases ?

Reply:
No, the second one should be the variable name. It is fixed now.